src/main/java/jenkins/scm/api/SCMSourceOwners.java

```diff
@@ -69,7 +70,7 @@
         public Iterator<SCMSourceOwner> iterator() {
             Jenkins j = Jenkins.getInstance(); // TODO 1.590+ getActiveInstance
```

Comment:
> We do wait for all pending HTTP requests to complete before terminating, right @stephenc? 

Well we don't because of how things work, but given that stapler's app will have been changed to `HudsonIsRestarting` before we even start the cleanUp() let alone clear the instance, any requests in flight are liable to blow up anyway, so having them blow up from this call is probably the lesser (more controlled) risk